README.md

```diff
@@ -37,22 +37,20 @@ The library supports _x86_ and _amd64_ architectures.
 ARM architecture is not supported.
 Please raise an issue to the library if you need ARM support && ready to provide proper test environment.
 
-:exclamation: It is **not recommended** to use WinP with _32bit_ Java on a _64bit_ operating system. 
-In such case the library will be running in the WoW64 mode; 
+:exclamation: It is **not recommended** to use WinP with _32bit_ Java on a _64bit_ operating system.
+In such case the library will be running in the WoW64 mode;
 and it will be unable to properly work with 64bit processes in the system.
 E.g. any process information query call (e.g. Environment variables retrieval) may fail if you run it against 64bit process.
 
 ## Supported Windows versions
 
 The current version of WinP is known to work correctly on the following Windows versions:
 
-* Windows XP SP2 and above
-* Windows Server 2003 and above
+* Windows 10 and above
+* Windows Server 2019 and above
```

Comment:
BTW do you know if this is dropping support for Windows Server 2016? I am definitely OK with dropping support for Windows Server 2012, since its [extended end date of October 2023](https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2012) has already passed, but I wonder if there may be Jenkins users still using Windows Server 2016. Jenkins users might still be using Windows Server 2016 since although its mainstream end date was in January of 2022, its [extended end date isn't until 2027](https://learn.microsoft.com/en-us/lifecycle/products/windows-server-2016). Keeping support for it would be ideal if it isn't prohibitively costly, but dropping support wouldn't be the end of the world — after all, we dropped support for CentOS 7 — but this would need to be communicated broadly to existing users (in the changelog and upgrade guide) and might be worth highlighting on the developer list in case others have a strong opinion about this.

Reply:
I am going to go ahead with this PR, but I am still hoping for a response to this comment, after which we can update the Release Notes/Upgrade Guide if necessary.

Reply:
I looked for information about which windows version was officially supported with default sdk for VS 2019, and found only Windows 10 and Server 2019. After all, Microsoft is trying to kill previous versions and force people to migrate, so we should not expect more than this.

Regarding our dependencies and windows functions used, I think it's pretty safe to say that Server 2016 should keep supported, as we don't use any "recent" API function.

Finally, people staying on a previous version of Windows probably prefer to stay on a previous version of Jenkins as well (it's more "stable" you know), so I would not worry too much about this.